samples/LowAllocationWebServer/LowAllocationWebServerLibrary/OwnedBuffer.cs

```diff
@@ -120,12 +127,12 @@ protected override void Dispose(bool disposing)
 
         public override Span<byte> GetSpan(int index, int length)
```

Comment:
Why do we need this method? What is the case where it's implementation is different from slicing `.Span`?